config/rbac/role.yaml

```diff
@@ -58,6 +58,7 @@ rules:
   resources:
   - consoleexternalloglinks
   - consolelinks
+  - consoleplugins
```

Comment:
I'm a bit confused about why we have those two role files now - added the missing resources to cluster-logging-operator_clusterrole.yaml just in case.